source/ios/AdaptiveCards/ADCIOSVisualizer/ADCIOSVisualizer/ViewController.m

```diff
@@ -512,6 +543,18 @@ - (void)keyboardWillBeHidden:(NSNotification *)aNotification
     self.enableCustomRendererButton = [self buildButton:@"Enable Custom Renderer" selector:@selector(toggleCustomRenderer:)];
     [layout[1] addArrangedSubview:self.enableCustomRendererButton];
 
+    if ([self appIsBeingTested])
+    {
+        // Set a background red color to signalize the app is in test mode
+        self.view.backgroundColor = [UIColor colorWithRed:1.0 green:0.80 blue:0.80 alpha:1];
+        
+        // Initialize the input results label
+        self.retrievedInputsTextView = [self buildLabel:@"" withIdentifier:@"SubmitActionRetrievedResults"];
+        
+        // Add the label to the container
+        [layout[2] addArrangedSubview:self.retrievedInputsTextView];
+    }
+    
```

Comment:
Consider pulling this out separately instead of inside `buildButtonsLayout`? As this view is a label with results, rather than a button.